README.md

```diff
@@ -18,24 +18,14 @@ Currently it is used by:
 It's based on:
 
 * [FSharp.Compiler.Service](https://github.com/fsharp/FSharp.Compiler.Service/) for F# language info.
-* [Ionide.ProjInfo](https://github.com/ionide/proj-info) for project/solution management.
-* [FSharpLint](https://github.com/fsprojects/FSharpLint/) for the linter feature.
-* [Fantomas](https://github.com/fsprojects/fantomas) for F# code formatting.
```

Comment:
Can you include these lines back? Other than that this looks great :)

Reply:
Idk how this happens 😅